### PR TITLE
Simplify EPEL installation

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: os_helper.bash [hunk 1]
```diff
@@ -139,13 +139,11 @@ tRHSubscribeAttach() {
 tRHEnableEPEL() {
   tIsRHEL || skip "Not required"
   if tIsRHEL 7; then
```
Comment (domcleal) [inline os_helper.bash]: Maybe just use `tSetOSVersion` and then `${OS_VERSION}` in the URLs, get rid of these conditionals.

Comment (ekohl) [inline os_helper.bash]: I noticed that EL6 used rpm where EL7 used yum so I thought to leave that in. Any reason they use different commands?

Comment (domcleal) [inline os_helper.bash]: I can't think of any reason.

Comment (ekohl) [inline os_helper.bash]: So what would be preferred? rpm? I think yum localinstall is needed on EL6 but don't know if it works on EL7.

Comment (domcleal) [inline os_helper.bash]: I think rpm's enough, I don't think yum's required for anything.

**Comment by ekohl** [comment]
Untested, but I think this should be it then.

**Comment by domcleal** [comment]
Thanks @ekohl!

**Pull request merged**